Formula/azure-cli.rb

```diff
@@ -446,8 +451,8 @@ class AzureCli < Formula
   end
 
   resource "PyYAML" do
-    url "https://files.pythonhosted.org/packages/4a/85/db5a2df477072b2902b0eb892feb37d88ac635d36245a72a6a69b23b383a/PyYAML-3.12.tar.gz"
-    sha256 "592766c6303207a20efc445587778322d7f73b161bd994f227adaa341ba212ab"
+    url "https://files.pythonhosted.org/packages/a8/c6/a8d1555e795dbd0375c3c93b576ca13bbf139db51ea604afa19a2c35fc03/PyYAML-4.2b4.tar.gz"
```

Comment:
note 3.13b1 is also an option if that's preferable

Reply:
Yes. But it comes out quite late after we build this release. The CLI in all other package management systems already were packed with 4.2b4. For the same of consistency, we will keep using 4.2b4 for now.

Thank you for reminding.